sites/fast-website/src/app/components/content-placement-container/content-placement-container.template.ts

```diff
@@ -1,29 +1,23 @@
 import { html, repeat, when } from "@microsoft/fast-element";
 import { ContentPlacementContainer } from "./content-placement-container";
+import { CommunityContentPlacementData } from "../../data/community.data";
 
 export const ContentPlacementContainerTemplate = html<ContentPlacementContainer>`
-    <div class="container">
+    <div
+        class="container ${x =>
+            x.section === "framework" ? "framework_container" : ""} ${x =>
```

Comment:
```suggestion
            ['framework', 'community'].includes(x.section) ? `${x.section}_container` : '' }
```

Reply:
I updated it to ${x => x.section}_container.

Reply:
That is a much better way of doing it. Thank you.